index.js

```diff
@@ -45,19 +47,26 @@ module.exports = function (content) {
         .replace(new RegExp(escapeRegExp(pathSep) + '+', 'g'), pathSep);
     var html;
 
-    if (content.match(/^module\.exports/)) {
+    if (content.match(/(?:^module\.exports)|(?:^export\s+default)|(?:^exports.default)/)) {
         var firstQuote = findQuote(content, false);
         var secondQuote = findQuote(content, true);
         html = content.substr(firstQuote, secondQuote - firstQuote + 1);
     } else {
         html = content;
     }
 
+    var exportsString = "module.exports = ";
```

Comment:
Would a simpler solution be to just keep the same export expression of the content? That way only `html-loader` would need to be configured.

I'm thinking something like:

```
var exportExprMatch = content.match(/^(module\.exports\s+=|export\s+default|exports.default\s+=)\s+/);
var exportExpr;
if (exportExprMatch) {
  var firstQuote = findQuote(content, false); 
  var secondQuote = findQuote(content, true);
  html = content.substr(firstQuote, secondQuote - firstQuote + 1);
  exportExpr = exportExprMatch[0];
} else {
  html = content;
  exportExpr = 'module.exports = ';
}

return "var path = '"+jsesc(filePath)+"';\n" +
        "var html = " + html + ";\n" +
        (requireAngular ? "var angular = require('angular');\n" : "window.") +
        "angular.module('" + ngModule + "').run(['$templateCache', function(c) { c.put(path, html) }]);\n" +
        exportExpr + "path;";
```

The README change would also then not be required.